src/components/VariantMutations.tsx

```diff
@@ -259,7 +259,7 @@ export const VariantMutations = ({ selector }: Props) => {
 
   return (
     <>
-      <div className='mb-8'>
+      <div className='mb-8' id='mutations'>
```

Comment:
Why do we need this id?

Reply:
The link form the mutations tab should lead to the mutations section on the main page, so we need it in order to use a hash link.

Reply:
It works, but only if the page is already fully loaded. When the page is still building up, I end up in the wrong place on the page.

Reply:
I don't know how to fix this or whether it is possible to fix this at all with simple means. I guess that the issue is that the page loads tons of data and builds up incrementally, thus it scroll to the "correct" place, but after loading the next diagram, the actually correct place is further down.
How much of a hard requirement is this? Could we simply leave out this feature? Or implement something else instead (only showing the targetted component)? I think we should at least not leave a non-working feature here.

Reply:
I set a timeout after which the scroll effect happens.